lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -61,82 +60,75 @@ def setUp(self):
                 'entrance_exam_enabled': True,
             }
         )
-        with self.store.bulk_operations(self.course.id):
```

Comment:
That prevents some updates to the modulestore until the end of the block. Atomic updates and all that. But a lot of tests actually do want updates as they go, particularly if they are saving the return values from ItemFactory - in that case, as they go, they do notably want the `children` of blocks to be updated as new items get added.

I don't know how things were working exactly in Old Mongo, but it's a lot less complicated and a lot more forgiving. I think since there isn't any versioning, objects were always live updated anyway? :shrug: